OWML.Common/Menus/IModPauseMenu.cs

```diff
@@ -2,7 +2,10 @@
 {
     public interface IModPauseMenu : IModPopupMenu, IModOWMenu
     {
-        IModTitleButton ResumeButton { get; }
+        IModTabbedMenu OptionsMenu { get; }
```

Comment:
Was this needed?

Reply:
yep, same for main menu. Not sure why it doesn't get resolved to IModOWMenu

Reply:
(yes, we now have a ton of compile warnings :shrug: